sitestats/sitestats-bundle/src/resources/Messages.properties

```diff
@@ -325,3 +325,6 @@ predefined_report5_description = Show users who have never visited the site.
 #
 predefined_report6_title = Users with no activity
 predefined_report6_description = Show users with no activity in site.
+
+format_datetime= MM/dd/yyyy hh:mm a
```

Comment:
It's incorrect to define these date formats in the Message.properties for each locale. Every framework should have a way to look it up from the locale.  getDateFormatString/getTimeFormatString in velocity, jsf (via InputDate) and RSF. Is the problem here that wicket doesn't have one yet in core? I think there is this that some tools that used sakai-wicket used, but this is wrong too (but at least easier to do a quick fix one place thant 5) https://github.com/sakaicontrib/SCORM.2004.3ED.RTE/blob/master/wicket-tool/src/java/org/sakaiproject/wicket/model/SimpleDateFormatPropertyModel.java